ogticrd/covid-safe-paths
# Feat: pagination bulletin screen

#### Description:


I added the pagination to bulletin screen

#### Linked issues:


Fix: [133](https://trello.com/c/hSK6NWEp/133-agregar-paginaci%C3%B3n-a-boletines)

#### Screenshots:



#### How to test:


- `git pull && git checkout Feat/133-pagination-bulletin-screen`
- Head over to bulletin screen and check that you have only 10 bulletins at the first time
- Press **VER MAS** button and it should display you 10 bulletins more
